kotlin/openID4VP/src/commonMain/kotlin/io/mosip/openID4VP/common/CborUtils.kt

```
@@ -15,22 +15,28 @@ import java.io.ByteArrayInputStream
 
+    return baos.toByteArray()
+}
+
+@Throws(Exception::class)
+fun encodeToCBOR(input: DataItem): ByteArray {
+    try {
+        val baos = ByteArrayOutputStream()
+        val encoder = CborEncoder(baos)
+        encoder.encode(input)
+        return baos.toByteArray()
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)